pipeline-scripts/buildlib.groovy

```diff
@@ -724,6 +724,27 @@ def build_ami(major, minor, version, release, yum_base_url, ansible_branch, mail
     }
 }
 
+def sweep(build_version, sweep_builds) {
```

Comment:
:heavy_check_mark: fixed!
I probably forgot I was doing Groovy, since I rely on the same mental clues ... I need to make the time to setup some linting for us, so humans won't need to spend time reviewing style.